statement.go

```diff
@@ -692,6 +696,27 @@ WHERE
 	}, nil
 }
 
+type TruncateTableStatement struct {
+	Table string
+}
+
+func (s *TruncateTableStatement) Execute(session *Session) (*Result, error) {
+	if session.InRwTxn() {
```

Comment:
I think it is better to check whether or not in `InRoTxn` because the behavior is a bit confusing.
```
> BEGIN RO;
...
> SELECT * FROM tbl;
...
1 rows in set (3 msecs)

> TRUNCATE TABLE tbl;

> SELECT * FROM tbl;
...
1 rows in set (3 msecs)
# User: not truncated? Why?
```

Reply:
That makes sense. Fixed in 34230379bba552ea6914679b0bd967a494733e76.